v5/pivotal/aggregator.go

```diff
@@ -0,0 +1,224 @@
+	aggregation := Aggregation{
+		aggregatedResponse: make(map[string]interface{}),
+		service:            a,
+	}
+	return &aggregation
+}
+
+func newAggregatorService(client *Client) *AggregatorService {
+	return &AggregatorService{client}
+}
+
+// Adds the url for the story to the aggregation data.
+func (a *Aggregation) Story(projectID, storyID int) *Aggregation {
+	a.requests = append(a.requests, BuildStoryURL(projectID, storyID))
+	return a
+}
+
+func (a *Aggregation) StoryUsingStoryID(storyID int) *Aggregation {
+	a.requests = append(a.requests, BuildStoryURLOnlyUsingStoryID(storyID))
+	return a
+}
+
+// Stories adds the urls for the slice of story IDs.
+func (a *Aggregation) Stories(projectID int, storyIDs []int) *Aggregation {
+	for _, storyID := range storyIDs {
+		a.Story(projectID, storyID)
+	}
+	return a
+}
+
+// Adds the url for the comments to the aggregation data.
+func (a *Aggregation) CommentsOfStory(projectID, storyID int) {
+	a.requests = append(a.requests, buildCommentsURL(projectID, storyID))
+}
+
+// Comments adds the requests for getting the comments of the stories in storiesToGet.
+func (a *Aggregation) CommentsOfStories(projectID int, storyIDs []int) *Aggregation {
+	for _, storyID := range storyIDs {
+		a.CommentsOfStory(projectID, storyID)
+	}
+	return a
+}
+
+// Adds the url for the reviews to the aggregation data.
+func (a *Aggregation) ReviewsOfStory(projectID, storyID int) *Aggregation {
+	a.requests = append(a.requests, buildReviewsURL(projectID, storyID))
+	return a
+}
+
+// Comments adds the requests for getting the comments of the stories in storiesToGet.
+func (a *Aggregation) ReviewsOfStories(projectID int, storyIDs []int) *Aggregation {
+	for _, storyID := range storyIDs {
+		a.ReviewsOfStory(projectID, storyID)
+	}
+	return a
+}
+
+func BuildStoryURLOnlyUsingStoryID(storyID int) string {
+	return fmt.Sprintf("/services/v5/stories/%d", storyID)
+}
+
+func BuildStoryURL(projectID, storyID int) string {
+	return fmt.Sprintf("/services/v5/projects/%d/stories/%d", projectID, storyID)
+}
+
+func buildCommentsURL(projectID, storyID int) string {
+	return fmt.Sprintf("/services/v5/projects/%d/stories/%d/comments", projectID, storyID)
+}
+
+func buildReviewsURL(projectID, storyID int) string {
+	return fmt.Sprintf("/services/v5/projects/%d/stories/%d/reviews?fields=id,story_id,review_type,review_type_id,reviewer_id,status,created_at,updated_at,kind", projectID, storyID)
+}
+
+func maxPagesPagination(total int, perPage int) int {
+	pagesNumber := math.Ceil(float64(total) / float64(perPage))
+	return int(pagesNumber)
+}
+
+func paginate(reqs []string, currentPage, total int, perPage int) []string {
+	firstEntry := (currentPage - 1) * perPage
+	lastEntry := firstEntry + perPage
+
+	if lastEntry > total {
+		lastEntry = total
+	}
+
+	return reqs[firstEntry:lastEntry]
+}
+
+// Sends the request for the aggregation.
+func (a *Aggregation) Send() (*Aggregation, error) {
```

Comment:
It seems like all of the other equivalent methods in the module return the `*http.Response` as well, e.g., [`List()`](https://github.com/oschwald/go-pivotaltracker/blob/c3d83abf398021f13ce6574c9b575387e1bd55ef/v5/pivotal/projects.go#L108). It'd probably make sense to do that for consistency. I am not sure the exact use case.